### DISCUSSION TITLE
Why does this project's enforcer settings block my local Nexus cache?

### DISCUSSION THREAD

**Discussion by InfoSec812** [quarkiverse/quarkus-openapi-generator#168] [Q&A]
I have a local Nexus 3 repository I use to cache my Maven artifacts (because crappy bandwidth), but when I try to compile this project it rejects my Maven settings:

```
[INFO] --< io.quarkiverse.openapi.generator:quarkus-openapi-generator-parent >--
[INFO] Building Quarkus - Openapi Generator - Parent 1.0.0-SNAPSHOT      [1/14]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-java-version) @ quarkus-openapi-generator-parent ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.BannedRepositories failed with message:
Current maven session contains banned repository urls, please double check your pom or settings.xml:
nexus - http://nexus3.cpl:9081/repository/maven-public/


Current maven session contains banned plugin repository urls, please double check your pom or settings.xml:
nexus - http://nexus3.cpl:9081/repository/maven-public/
```

**Comment by hbelmiro** [comment]
@ricardozanini I think you're the best person to answer this.

**Comment by ricardozanini** [comment]
@InfoSec812 this enforcer rule is defined by the Quarkus Plugin. Any other plugins in quarkiverse behave similarly. @gastaldi, am I missing something? Is there a way to override this behavior?

**Reply by gastaldi** [reply]
AFAIR Maven 3.8.x no longer accepts HTTP repositories. Can you enable HTTPS in your local Nexus3 repository?

**Reply by ricardozanini** [reply]
@InfoSec812 ^

**Reply by InfoSec812** [reply]
> AFAIR Maven 3.8.x no longer accepts HTTP repositories. Can you enable HTTPS in your local Nexus3 repository?

@gastaldi Not easily. It's running as a Podman container on my local machine because my internet connection is not good and not reliable.
